packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -90,9 +90,53 @@ function postToWhistler(webex, type, payload) {
     }));
 }
 
-describe.skip('plugin-calendar', () => {
+function postNotesToWhistler(webex, type, payload) {
+  return retry(() => webex.request({
+    method: 'POST',
+    service: 'whistler',
+    resource: 'calendarEvent',
+    body: payload,
+    qs: {
+      changeType: type,
+      useProduction: typeof process.env.WDM_SERVICE_URL === 'undefined' || process.env.WDM_SERVICE_URL.includes('wdm-a.wbx2.com')
+    }
+  })
+    .then((res) => {
```

Comment:
Minor note:
```suggestion
    .then(({body: createdMeeting}) => {
```
this would replace the next line as well

Reply:
I have to return `res` from the `then()`, so I'll have to make a change in the return statement as well

Reply:
The first thing you do in the `then` statement following this is grab the `body` property and assign it to `createdMeeting`. You could skip that step.